### PR TITLE
Document vagrant build errors

### PR DESCRIPTION
**Pull request by cantino**
Building in a shared folder in vagrant causes errors that are very hard to diagnose.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,6 +87,8 @@ The full build process will take a while.  Running on a Linux VM, it took about
 twenty minutes.  Once this build process has completed once, you can run other BuildRoot commands to rebuild only certain sections
 and it will go much more quickly (<5 min).
 
+> Note: if you're using vagrant, as with the Kubos SDK, do not use a shared folder for the kubos-linux-build or buildroot directories, as the build process uses hardlinks that error with vagrant's shared folders.
```
Comment (CatGarab) [inline README.md]: nit: Capitalize 'If' and 'Vagrant'

Comment (CatGarab) [inline README.md]: Add an additional note that certain scripts expect `kubos-linux-build` and `buildroot-2019.02.2` to be peer (sibling?) directories

Comment (cantino) [reply README.md]: I believe we say that above?

Comment (cantino) [reply README.md]: `These components should be setup as children of the same parent directory`

Comment (CatGarab) [reply README.md]: oh hey, so we do :laughing:

**Pull request merged**